lib/livebook/config.ex

```diff
@@ -687,28 +687,37 @@ defmodule Livebook.Config do
   @doc """
   Parses zero trust identity provider from env.
   """
-  def identity_provider!(env) do
-    case System.get_env(env) do
+  def identity_provider!() do
```

Comment:
Oh, sorry, I know how we can test it. Keep the exact same code as before, but do this:

```
System.put_env("LV_TEST_IP", "custom:...")
identity_provider!("LV_TEST_IP")
```

We can basically keep the same code as it was (only the one line of code change bug fix).